src/components/common/Button.stories.ts

```diff
@@ -0,0 +1,33 @@
+import { Meta } from "@storybook/react";
+import Button from "./Button";
+
+const meta: Meta<typeof Button> = {
+  title: "Common/Button",
+  component: Button,
+  tags: ["autodocs"],
+  parameters: {
+    docs: {
+      description: {
+        component: "ボタンコンポーネント",
```

Comment:
```suggestion
        component: "ログイン/サインアップフォーム用ボタン",
```
ここにはコンポーネントの具体的な概要を書いた方が良いと思うので、こんな感じの方がベターかもです